app/instance-initializers/global.js

```diff
@@ -0,0 +1,10 @@
+export function initialize(appInstance) {
```

Comment:
Note the commit message. I'd welcome a sanity check from someone more familiar with addons.
https://github.com/CenterForOpenScience/exp-addons/pull/11